source/language.c

```diff
@@ -795,4 +795,104 @@ const char* const g_strings[StrId_Max][16] =
 			"已完成 %zu / %zu KiB"
 		),
 	},
+
+	[StrId_NetSender] =
+	{
+		STR_EN("3dslink NetSender"),
```

Comment:
Minor terminology nitpick: I think I would call this functionality a "3dslink server" instead of a "netsender". I don't think I've ever seen anything called a "netsender".